<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxiesConfigWidget</class>
 <widget class="QWidget" name="ProxiesConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeView" name="ProxiesList_"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="AddProxyButton_">
       <property name="text">
        <string>Add proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="UpdateProxyButton_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RemoveProxyButton_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Remove proxy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditUrlsButton_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit URLs</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EditListsButton_">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit scripts</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ProxiesList_</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
